src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        ColorSupplier supplier = new ColorSupplier();
```

Comment:
this logic related to FigureSupplier class. You should create `public Figure getRandomFigure()` in FigureSupplier